PowerShell/JumpCloud Module/Public/Commands/Get-JCCommand.ps1

```diff
@@ -1,10 +1,34 @@
 
 Function Get-JCCommand ()
 {
-    [CmdletBinding(DefaultParameterSetName = 'ReturnAll')]
+    [CmdletBinding(DefaultParameterSetName = 'SearchFilter')]
 
     param
     (
+
+    #### NEW PARAMS
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'The command to execute on the server.')]
+        [String]$command,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Name of the command')]
+        [String]$name,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Command Type')]
```

Comment:
"The command type of the JumpCloud Command you wish to search for."

And I think we should add the validate set here:
[ValidateSet('windows', 'mac', 'linux')]